src/layouts/entity.tsx

```diff
@@ -56,8 +59,15 @@ const EntityLayout = ({ data }: EntityLayoutProps) => {
   const showInsights = insights?.title && insights?.insights;
   const showReviews = reviews?.title && reviews?.reviews;
 
+  const { t } = useTranslation();
+  const count = Math.ceil(Math.random() * 2);
+
   return (
     <>
+      <Trans>Welcome to My House</Trans>
+      <h1>{t("Welcome to React")}</h1>
+      <Trans count={count}>You have {{ count }} messages.</Trans> (Refresh to
```

Comment:
Are these two use cases supported?
1. Using JSX in replacements: https://github.com/yextconsulting/pages-starter-react-consulting/compare/jronkin/i18n..jronkin/translations#diff-3d90ccc767139e6a4893cc2a8db9cce23e38307d12bd763c661d880c92651cf8R24
2. Using replacements in places where we can't use JSX, i.e. the `t("Text {{ count }}")` form -- would that just have to be `t("Text {{ count }}").replaceAll("{{ count }}", count)`?

Reply:
Added examples for each of these use cases.